tests/test_signal.py

```diff
@@ -344,6 +344,44 @@ def test_find_nearest_frequency():
     npt.assert_allclose(actual, expected)
 
 
+def test_reshape():
+
+    # test reshape
+    x = np.random.rand(6, 256)
+    signal_in = Signal(x, 44100)
+    signal_out = signal_in.reshape((3, 2))
+
+    npt.assert_allclose(signal_in._data.reshape(3, 2, -1), signal_out._data)
+    assert id(signal_in) != id(signal_out)
+
+    # test assertion for non-tuple input
+    with pytest.raises(ValueError):
```

Comment:
Additional check for new shape of type int?

Reply:
yes, good idea - I've done that :)